package.json

```diff
@@ -67,7 +67,8 @@
     "node": ">=8.11.0"
   },
   "dependencies": {
-    "@tensorflow/tfjs": "0.13.3",
+    "@tensorflow/tfjs": "0.15.1",
+    "build": "^0.1.4",
```

Comment:
I think this is a mistake while resolving the merge conflict. I removed it in develop.